docs/src/pages/[platform]/connected-components/storage/storageimage/examples/DefaultStorageImageExample.tsx

```diff
@@ -1,5 +1,5 @@
 import { StorageImage } from '@aws-amplify/ui-react-storage';
 
 export const DefaultStorageImageExample = () => {
-  return <StorageImage alt="cat" imgKey="cat.jpg" accessLevel="guest" />;
+  return <StorageImage alt="sleepy-cat" path="guest/cat.jpg" />;
```

Comment:
I see we don't have an example for this today, but can we add an example of using a protected / private image along with the callback?

Reply:
Yeah should I show a full example with fetchAuthSession or just assume they have the identityId?

Reply:
For the new API they just use the callback version which will provide the identityId, right?

Reply:
Lol, yes